Lib/idlelib/editor.py

```diff
@@ -815,32 +842,43 @@ def set_notabs_indentwidth(self):
                                                   type='int')
 
     def reset_help_menu_entries(self):
-        "Update the additional help entries on the Help menu"
+        """Update the additional help entries on the Help menu.
+
+        First the existing additional help entries are removed from
```

Comment:
In this doc-string, I find the added description of how the method works entirely unnecessary, because it is already similarly detailed in comments.